docs/devices-api/api-reference/user/revoke-provider-access.mdx

```diff
@@ -1,15 +1,17 @@
 ---
 title: "Revoke Provider Access"
 description: "Revokes the access token for a given provider."
-api: "DELETE /user/revoke"
+api: "DELETE /user/:id/revoke"
 ---
 
-## Query Params
+## Path Params
 
-<ParamField query="userId" type="string" required>
+<ParamField path="userId" type="string" required>
```

Comment:
Ideally we would name the path param field here with the same name we have on the URL on the `api` above - `api: "DELETE /user/:id/revoke"`